tensorflow/compiler/xla/service/cpu/simple_orc_jit.cc

```diff
@@ -102,9 +107,15 @@ llvm::StringRef GetHostCpuName() {
 
 CompilerFunctor::VectorIntrinsics GetAvailableIntrinsics() {
   CompilerFunctor::VectorIntrinsics intrinsics;
+#ifdef __SSE4_1__
   intrinsics.sse_intrinsics = (&__xla_cpu_runtime_ExpV4F32SSE != nullptr);
```

Comment:
Checking for nullptr should no longer be necessary -- just set `intrinsics.sse_intrinsics` to `true` guarded under `#ifdef __SSE4_1__`.

Reply:
will do.